src/library/Cardano/CoinSelection/Fee.hs

```diff
@@ -98,20 +98,41 @@ newtype Fee = Fee { unFee :: Coin }
     deriving stock (Eq, Generic, Ord)
     deriving Show via (Quiet Fee)
 
--- | Defines the maximum size of a dust coin.
+-- | Defines the /maximum/ size of a __dust coin__.
 --
--- Change values that are less than or equal to this threshold will not be
--- included in coin selections produced by the 'adjustForFee' function.
+-- Functions that accept a 'DustThreshold' argument will generally exclude
+-- values that are /less than or equal to/ this threshold from the 'change'
+-- sets of generated selections, /coalescing/ such coins together into larger
```

Comment:
not `change` set ?

Reply:
> not `change set` (rather than sets) ?

Good question. Since we are talking about "selection**s**" (plural) generated by a function (being applied many times), we must also use the plural form "set**s**" here.